api/src/permissions/permissions.service.spec.ts

```diff
@@ -0,0 +1,262 @@
+    });
+
+    it("can create a Group Map", () => {
+        expect(Object.keys(groupMap).length).toBe(8);
+    });
+
+    it("can calculate inherited groups", () => {
+        const res = Group.getAccess(
+            ["group-super-admins"],
+            groupMap,
+            [DocType.Language],
+            AclPermission.View,
+        );
+
+        expect(res.includes("group-languages")).toBe(true);
+    });
+
+    it("can update inherited groups", () => {
+        Group.updateGroups(
+            [
+                {
+                    _id: "group-languages",
+                    type: "group",
+                    updatedTimeUtc: 3,
+                    name: "Languages",
+                    acl: [
+                        {
+                            type: "language",
+                            groupId: "group-public-content",
+                            permission: ["view"],
+                        },
+                        {
+                            type: "language",
+                            groupId: "group-private-content",
+                            permission: ["view", "translate"],
+                        },
+                        {
+                            type: "language",
+                            groupId: "group-public-editors",
+                            permission: ["view"],
+                        },
+                        {
+                            type: "language",
+                            groupId: "group-private-editors",
+                            permission: ["view", "translate"],
+                        },
+                    ],
+                },
+            ],
+            groupMap,
+        );
+
+        const res1 = Group.getAccess(
+            ["group-private-content"],
+            groupMap,
+            [DocType.Language],
+            AclPermission.Translate,
+        );
+        const res2 = Group.getAccess(
+            ["group-public-editors"],
+            groupMap,
+            [DocType.Language],
+            AclPermission.Translate,
+        );
+
+        expect(res1.includes("group-languages")).toBe(true);
+        expect(res2.includes("group-languages")).toBe(false);
+    });
+
+    it("can remove a group", () => {
+        Group.removeGroups(["group-languages"], groupMap);
+
+        const res = Group.getAccess(
+            ["group-super-admins"],
+            groupMap,
+            [
+                DocType.Audio,
+                DocType.Change,
+                DocType.Content,
+                DocType.Group,
+                DocType.Image,
+                DocType.Language,
+                DocType.MediaDownload,
+                DocType.Post,
+                DocType.Tag,
+                DocType.User,
+                DocType.Video,
+            ],
+            AclPermission.View,
+        );
+
+        expect(res.includes("group-languages")).toBe(false);
+    });
+
+    it("can remove an ACL", () => {
```

Comment:
I suggest adding a comment on the item that was removed from the list to make it more clear.

Reply:
Perhaps check the others tests as well if a comment would be helpful there.